src/distributions/dragonwell/installer.ts

```diff
@@ -0,0 +1,173 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+
+import {JavaBase} from '../base-installer';
+import {extractJdkFile, getDownloadArchiveExtension} from '../../util';
+import {IDragonwellVersions, IDragonwellAllVersions} from './models';
+import {
+  JavaDownloadRelease,
+  JavaInstallerOptions,
+  JavaInstallerResults
+} from '../base-models';
+
+export class DragonwellDistribution extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions) {
+    super('Dragonwell', installerOptions);
+  }
+
+  protected async findPackageForDownload(
+    version: string
+  ): Promise<JavaDownloadRelease> {
+    if (!this.stable) {
+      throw new Error('Early access versions are not supported');
+    }
+    let majorVersion = version;
+    if (version.includes('.')) {
+      const splits = version.split('.');
+      majorVersion = splits[0];
+      version = splits.length >= 3 ? splits.slice(0, 3).join('.') : version;
+    }
+    const edition = majorVersion == '17' ? 'Standard' : 'Extended';
```

Comment:
Could you please describe what is the difference between Standard and Extended version of the JDK. Also, I see that all versions have Standard and Extended editions but you have limited 17 to Standard only.

Reply:
> Could you please describe what is the difference between Standard and Extended version of the JDK. Also, I see that all versions have Standard and Extended editions but you have limited 17 to Standard only.

- Alibaba Dragonwell Standard Edition: OpenJDK upstream based and have more enhancements, including bug-fixing, security patches, tooling support, etc.
- Alibaba Dragonwell Extended Edition: everything in the standard edition, plus: extra customized/significant features optimized for the cloud and widely used in Alibaba's production environment.
And dragonwell17 only has standard edition.